store.json

```diff
@@ -76,5 +76,14 @@
     "name": "certinfo",
     "network": "func_functions",
     "repo_url": "https://github.com/stefanprodan/openfaas-certinfo"
+  },
+  {
+    "icon": "https://raw.githubusercontent.com/stefanprodan/openfaas-promq/master/logo.png",
+    "title": "Prometheus query runner",
+    "description": "Runs PromQL for a given Prometheus server. Documentation: https://github.com/stefanprodan/openfaas-promq",
```

Comment:
IMO you can remove the part after the `server.` because it's just the repo URL.

Looks good otherwise 👍

Reply:
The repo URL doesn't show in the store UI, how will people know how to call the function?